app/grandchallenge/reader_studies/tasks.py

```diff
@@ -46,6 +46,27 @@ def add_scores(*, instance_pk, pk_set):
             add_score(instance, ground_truth.answer)
 
 
+@shared_task
```

Comment:
Please do not use the default queue. Please ensure that any new tasks are idempotent and use one of the acks-late queues (depending on their memory/duration requirements).

Reply:
I think the micro-short queue should be sufficient for this